modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -2100,6 +2099,71 @@ public void testExecuteObjectActionWithUsePreferredLanguageForGuestsParameter()
 		_userLocalService.deleteUser(user);
 	}
 
+	@Test
+	public void testFillLabelMap() throws Exception {
+
+		// label map as null
+
+		Locale locale = _objectDefinition.getDefaultLocale();
+
+		ObjectAction objectAction1 = _addObjectAction(
+			StringPool.BLANK, null, "objectAction1Name",
+			ObjectActionExecutorConstants.KEY_GROOVY,
+			ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE,
+			new UnicodeProperties(), false);
+
+		Assert.assertEquals(
+			Collections.singletonMap(locale, "objectAction1Name"),
```

Comment:
@pedro-oliveira446 define `locale` inline and replace `"objectAction1Name"` with `objectAction1.getName()`